app/services/foreman/access_permissions.rb

```diff
@@ -312,12 +312,13 @@
     pc_ajax_actions = [:parameters]
     subnets_ajax_actions = [:freeip]
     tasks_ajax_actions = [:show]
+    template_kinds =  [:PXELinux, :PXEGrub, :iPXE, :provision, :finish, :script, :user_data, :ZTP]
```

Comment:
could you deduplicate this list? https://github.com/theforeman/foreman/blob/develop/db/seeds.d/07-config_templates.rb#L9

just put it into a constant somewhere in TemplateKind and use it in both places?